org.lyx.LyX.appdata.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2012-2023 lyx.org -->
+<component type="desktop">
+  <id>org.lyx.LyX</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-or-later</project_license>
+  <name>LyX</name>
+  <summary>High level LaTeX frontend</summary>
+  <description>
+    <p>
+      LyX combines the power and flexibility of TeX/LaTeX with the ease 
+      of use of a graphical interface. This results in world-class support 
+      for creation of mathematical content (via a fully integrated equation 
+      editor) and structured documents like academic articles, theses, and books.
+    </p>
+    <p>
+      LyX is for people who want their writing to look great, right out 
+      of the box. No more endless tinkering with formatting details, 
+      “finger painting” font attributes or futzing around with page boundaries. 
+      You just write.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://www.lyx.org/images/LGT/start.png</image>
+      <caption>The startup screen of LyX</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://www.lyx.org/images/about/main_window.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://www.lyx.org/images/LGT/cross-reference-inserted.png</image>
+    </screenshot>
+  </screenshots>
+  <categories>
+    <category>Science</category>
+    <category>Education</category>
+  </categories>
+
+
+  <url type="bugtracker">https://www.lyx.org/MailingLists</url>
+  <url type="homepage">https://www.lyx.org</url>
+  <url type="help">https://wiki.lyx.org/</url>
+  <developer_name>LyX</developer_name>
+  <kudos>
+    <kudo>HiDpiIcon</kudo>
+    <kudo>HighContrast</kudo>
+    <kudo>UserDocs</kudo>
+  </kudos>
+  <launchable type="desktop-id">lyx.desktop</launchable>
```

Comment:
This should be `$FLATPAK_ID.desktop`

Reply:
`rename-desktop-file` handles it. It's fine.

Reply:
Oh, yea, didn't notice that.